docs/configuration.md

```diff
@@ -179,6 +179,15 @@ of the most common options to set are:
     (e.g. <code>2g</code>, <code>8g</code>).
   </td>
 </tr>
+<tr>
+ <td><code>spark.executor.pyspark.memory</code></td>
+  <td>Not set</td>
+  <td>
+    The amount of memory to be allocated to PySpark in each executor, in MiB
```

Comment:
We should probably mention that this is added to the executor memory request in Yarn mode.

Reply:
I've added "When PySpark is run in YARN, this memory is added to executor resource requests."